hs_core/views/resource_rest_api.py

```diff
@@ -740,7 +739,6 @@ def put(self, request, pk):
                                                                          hydroshare_host=domain)
                 # Update resource metadata
                 rm.write_metadata_to_resource(resource, update_title=True, update_keywords=True)
-                create_bag_files(resource)
```

Comment:
Should your new create_bag() method need to go here in place of create_bag_files()? Otherwise, will this REST API endpoint still work?

Reply:
I am not really sure what this endpoint is useful for in light of fact that we decided in a meeting a few weeks ago to not even write out the metadata files (resourcemetadata.xml) any longer. I left it alone as much as I could in hopes of refactoring it later.

Reply:
@ismagilani OK. Got it. Then it'd be good to put in a comment there indicating this and the refactoring need to be done later, so it does not slip through the cracks.